web/src/lib/components/shared-components/side-bar/side-bar.svelte

```diff
@@ -108,6 +109,13 @@
 			isSelected={$page.route.id === '/(user)/explore'}
 		/>
 	</a>
+	<a data-sveltekit-preload-data="hover" href={AppRoute.MAP} draggable="false">
+		<SideBarButton
+			title="Map"
+			logo={Map}
+			flippedLogo={true}
```

Comment:
Does this icon need to be flipped?

Reply:
I just copy-pasted this :P
Yeah, I'll remove that